Scripts/network-wranglers.js

```diff
@@ -0,0 +1,10 @@
+export function loadTextFile(path){
```

Comment:
Is there a reason to use this over jquery AJAX?

Reply:
Not especially. I was scared of using a powerful library that does lots of things, just to do one thing. E.g. in C++ I've been warned against using Boost unless totally necessary because "before you know it everything will depend on Boost and it will become impossible to get rid of". But lacking experience in JS / jquery, I'll defer to you.